### PR TITLE
doc(English): add docs of English version

### PR DESCRIPTION
**Pull request by Adek06**

### CODE REVIEW

**Comment by Adek06** [comment]
临时查看地址：http://139.9.239.23:4000/en_US/

File: en_US/deployments/create_deployment.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Create a new deployment
+In this tutorial, you will learn how to create a new EMQ X Cloud deployment. To learn how to delete an existing EMQ X Cloud deployment, please refer to [Delete Deployment](./delete_deployment.md).
+
+## limitation
+
```
Comment (zibuyu1995) [inline en_US/deployments/create_deployment.md:5]: * Before creating a deployment, your account overdraft limit should be greater than the 24-hour deployment cost. You can go to the [Price Estimate](https://cloud.emqx.io/console/deployments/0?oper=new) page to estimate the 24-hour deployment cost, or submit a [tickets](https://cloud.emqx.io/console/tickets) to increase the account overdraft limit.
* There are no unpaid bills in your account.

File: en_US/deployments/create_deployment.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+* Before creating a deployment, your account balance cannot be less than the total cost of the created deployment for **24** hours. You can go to [Price Estimate](https://cloud.emqx.io/console/deployments/0?oper=new) to view the hourly cost of the selected deployment specification * 24, and you can go to [Payment Overview Page](https://cloud.emqx.io/console/billing/overview)  to complete the account balance recharge.
+
+  > The deployment specification is 1000 lines, 100 tps, and the hourly price is 1.17 Yuan, then your account balance should be greater than **28.08** USD (1.17 * 24)
```
Comment (zibuyu1995) [inline en_US/deployments/create_deployment.md:8]: 这里每小时价格好像不对
The deployment specification is 1000 lines, 100 tps, and the hourly price is 1.17 USD, then your account overdraft should be greater than **28.08** USD (1.17 * 24)

File: en_US/billing/overview.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+  ![Account Balance](./_assets/account_balance.png)
+  
+* PayPal Payment
+  ![PayPal Payment](./_assets/paypal_payment.png)
+  
+* Other Payment Methods
+  
+  Through [tickets](https://cloud.emqx.io/console/tickets), you can pay using a:
+  * Currency other than USD.
+  * Method other than a credit card or PayPal.
```
Comment (zibuyu1995) [inline en_US/billing/overview.md:27]: Method other than a  PayPal.

File: en_US/faq.md [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+For detailed billing rules, you can view [Product Pricing](pricing.md)
+
+### How to payment?
+
+In the EMQ X Cloud console, click the `Payment` button at the account balance of `Billing` -> `Overview`, and complete the monthly bill payment.
+
+Currently, EMQ X Cloud supports `PayPal` to complete the monthly bill payment.
+
+### How to get the voucher?
+
```
Comment (zibuyu1995) [inline en_US/faq.md:91]: How to get the credits?

全局搜一下 voucher -> credits

**Review by zibuyu1995 — CHANGES_REQUESTED** [review]

File: en_US/deployments/create_deployment.md (hunk 2 above)
Comment (Adek06) [reply en_US/deployments/create_deployment.md]: 应该是 1000 lines, 1000 tps。价格以图上显示的为准吧。

File: en_US/README.md [hunk 5]
```diff
@@ -0,0 +1,169 @@
...
+  - Save data to PostgreSQL
+  - Save data to Cassandra
+  - Save data to MongoDB
+  - Save data to DynamoDB
+  - Save data to Redis
+  - Save data to ClickHouse
+  - Save data to TDengine
+  - Save data to OpenTSDB
+  - Save data to TimescaleDB
+- Save data to InfluxDB
```
Comment (oucb) [inline en_US/README.md:84]: 前面缺少空格

File: en_US/README.md [hunk 6]
```diff
@@ -0,0 +1,169 @@
...
+  
+- **Bridge data to Kafka, RabbitMQ and other messaging and stream processing platforms**
+  
+  - Bridge data to Kafka
+  - Bridge data to Pulsar
+  - Bridge data to RocketMQ
+  - Bridge data to RabbitMQ
+- Bridge data to RPC service
```
Comment (oucb) [inline en_US/README.md:92]: 前面缺少空格

**Review by oucb — CHANGES_REQUESTED** [review]

**Pull request merged**